README.md

```diff
@@ -120,3 +135,295 @@ This will return the following runtime images with the following names: `action-
 ```
 ./gradlew :tests:test
 ```
+
+---
+
+## Using Docker <a name="using_docker"></a>
+
+### Pre-requisites
+- [Docker](https://www.docker.com/)
+- [curl](https://curl.se/), [wget](https://www.gnu.org/software/wget/), or [Postman](https://www.postman.com/)
+
+
+0. Choose/create a folder of your liking and make sure docker daemon is running
```

Comment:
Do not use numbered bullets where possible. Instead this can become a lower section title like "Clone the repository" under the "Build" subsection of "Using Docker"